subsys/bluetooth/mesh/Kconfig

```diff
@@ -1457,30 +1457,15 @@ endmenu # Proxy
 choice BT_MESH_CRYPTO_LIB
 	prompt "Crypto library:"
 	default BT_MESH_USES_TFM_PSA if BUILD_WITH_TFM
-	default BT_MESH_USES_TINYCRYPT
+	default BT_MESH_USES_MBEDTLS_PSA
 	help
 	  Crypto library selection for mesh security.
 
-config BT_MESH_USES_TINYCRYPT
```

Comment:
I guess this should be moved back and marked as [DEPRECATED]

Reply:
Agree, and a release notes should be added that this option is deprecated.

Reply:
Ok, this should be addressed now